2.6.md

```diff
@@ -2740,7 +2740,7 @@ This object is associated with a `SupplyChain` object as an array of nodes. Thes
   <tr>
     <td><code>rid</code></td>
     <td>string</td>
-    <td>The OpenRTB RequestId of the request as issued by this seller.</td>
+    <td>The OpenRTB RequestId of the bid request as issued by the advertising system associated with this Supply Chain Node.</td>
```

Comment:
I recommend putting this through separately. I have more questions about the need to change this one, since it's possible vendors are relying on the previous definition.

Reply:
Can we just mark schain rid as deprecated? Is it not completely redundant?